src/vumi2/transports/smpp/processors.py

```diff
@@ -360,7 +360,7 @@ async def _handle_deliver_sm_esm_class(
         if esm_class.type == EsmClassType.DEFAULT:
             return False, None
 
-        content = pdu.params["short_message"].decode()
+        content = pdu.params["short_message"].decode("latin1")
```

Comment:
Can you add a comment explaining why we use `latin1` here, please? Perhaps something like this:
```
# The SMPP spec doesn't mention encodings at all for delivery reports, so assume they're plain ASCII and decode with latin1 to avoid decode errors.
```